modules/module-headings.php

```diff
@@ -142,6 +142,11 @@ function jetpack_get_module_i18n( $key ) {
 				'description' => _x( 'Publish posts by sending an email', 'Module Description', 'jetpack' ),
 			),
 
+			'private' => array(
```

Comment:
These changes are going to be overriden since this file is automatically created during the build task. The module heading in the `modules/module-slug.php` file is the source of the data. We don't have to remove it from the PR, but the changes here won't go into a live site if different than the module headers.